parfive/config.py

```diff
@@ -143,6 +143,14 @@ class SessionConfig:
     """
     env: EnvConfig = field(default_factory=EnvConfig)
 
+    done_callback: Optional[Callable] = None
```

Comment:
If you are wanting to use this to address #122 then I think that it should be possible to specify a list of callbacks, so that we can add some default ones and the users can add custom ones.

Reply:
Oh good point, I'll make it a list.